### DISCUSSION TITLE
Snippets support in markor

### DISCUSSION THREAD

**Discussion by harshad1** [gsantner/markor#1610] [Ideas]
IMO, we should consider some sort of snippet support in markdown. This will really help with things like YAML front matter etc

My initial thought is that we could have one block of text per file which you can then cat into a file at the cursor position via an action. A slightly more powerful system could include replacement of placeholders like the time dialog.

**Comment by harshad1** [comment]
I have a pr up #1624 

Snippets support interpolation of dates. We can add other features later, if needed

**Comment by gsantner** [comment]
issue --> #437
